Apps/WebClient/src/ClientApp/src/components/report/immunizationHistory.vue

```diff
@@ -150,22 +150,30 @@ export default class ImmunizationHistoryReportComponent extends Vue {
             key: "date",
             thStyle: { width: "15%" },
             thClass: this.headerClass,
+            thAttr: { "data-testid": "immunizationDateTitle" },
+            tdAttr: { "data-testid": "immunizationItemDate" },
```

Comment:
Suggest consistency in the naming for this throughout - either `immunizationItemX` & `immunizationTitleX` or `immunizationXTitle` & `immunizationXDate`.  Same goes for the `thAttr` and `tdAttr` properties in other files.

Reply:
Agreed.